packages/expo-video/android/src/main/java/expo/modules/video/VideoModule.kt

```diff
@@ -261,6 +264,28 @@ class VideoModule : Module() {
           }
         }
 
+      Property("bufferedPosition")
+        .get { ref: VideoPlayer ->
+          // Same as currentTime
+          runBlocking(appContext.mainQueue.coroutineContext) {
+            if (ref.player.currentMediaItem == null) {
+              return@runBlocking -1
+            }
+            if (ref.player.playbackState == STATE_BUFFERING) {
+              return@runBlocking 0
+            }
+            ref.player.bufferedPosition / 1000f
+          }
+        }
```

Comment:
I think we should make it asynchronous. It doesn't make sense to block both threads when one can run.

Reply:
@lukmccall Is it possible to have asynchronous properties?

Reply:
No, properties cannot be asynchronous (actually they could if they are promises, but it just doesn't make much sense). I think @lukmccall meant that this should be an async function instead of a property.

Reply:
Yes, I meant to replace it with an async function. Sorry, I should make it more clear 😅

Reply:
I'm not sure of this approach, we already block the thread for `currentTime` `currentLiveTimestamp` `currentOffsetFromLive`  properties, I know it's bad, at least the api would be more consistent (also the thread is locked for a very short time)